lib/secure-boot-lib.robot

```diff
@@ -82,6 +82,8 @@ Reset To Default Secure Boot Keys
     Enter Submenu From Snapshot    ${advanced_menu}    Reset to default Secure Boot Keys
     Read From Terminal Until    Are you sure?
     Press Enter
+    # QEMU needs a lot of time, why?
```

Comment:
@krystian-hebel Can it be the same reason as long boot time, or not really?
If not, we would need a second issue for that.

Reply:
It is the same, it worked fine with LAPIC. I'd still leave some delay, there are cases where this keyword is immediately followed by a power-off, I'm surprised it didn't cause any issues earlier.